blob/blob.go

```diff
@@ -401,7 +402,10 @@ func (b *Bucket) As(i interface{}) bool {
 
 // ReadAll is a shortcut for creating a Reader via NewReader with nil
 // ReaderOptions, and reading the entire blob.
-func (b *Bucket) ReadAll(ctx context.Context, key string) ([]byte, error) {
+func (b *Bucket) ReadAll(ctx context.Context, key string) (_ []byte, err error) {
+	ctx = trace.StartSpan(ctx, "gocloud.dev/blob.ReadAll")
+	defer func() { trace.EndSpan(ctx, err) }()
+
```

Comment:
Add to `List` as well.

Reply:
List makes no RPCs, and the returned ListIterator has no Stop or Close method, so we don't know when to end the span.